db/seeds.rb

```diff
@@ -5,162 +5,100 @@
+  },{
+    name: 'Pinshape', address: '210 - 128 West Hastings Street Vancouver, British Columbia V6B1G8 Canada', overview: 'Pinshape is the next generation 3D printing community & marketplace for brands, designers and makers. We specialize in making it easy to find and successfully print great 3D designs, while mitigating intellectual property risks through secure streaming technology.', employee_count: 10, tech_team_size: 10, website: 'https://pinshape.com/', logo: 'https://pbs.twimg.com/profile_images/499763216981835776/0oPskR_u_400x400.jpeg', twitter: 'pinshape', published: false
+  },{
+    name: 'Picatic', address: '33 Water St #205, Vancouver, BC V6B 1A1, Canada', overview: '"We help brands and publishers create great content. Follow us for insights on content marketing, publishing and more.
+
+    Every brand has distribution channels (social, blogs, newsletters). And every brand needs content to fill these channels. Quietly helps brands and publishers develop data-driven content that focuses on timeless topics to be promoted (and re-promoted) to increase reach, traffic, and revenue. Using our proprietary tools & analytics, keyword trends, and competitive landscape research Quietly helps shape evergreen content marketing opportunities through a thoughtful and strategic approach."', employee_count: 35, tech_team_size: 35, website: 'https://www.quiet.ly/', logo: 'https://pbs.twimg.com/profile_images/647542416585592833/tNOlcaYJ_400x400.png', twitter: 'quietlyapp', published: false
+  },{
+    name: 'Human API', address: '1402 - 409 Granville Street, Vancouver, BC V6C 1T1, Canada', overview: '"Human API is the easiest way to integrate health data from anywhere.
+
+    We provide a single, secure connection to data from medical records, wearable sensors, testing services, and wellness devices and apps. Using our end-user authentication process, individuals can securely share their health data with any application or system, regardless of how that data was recorded, processed or stored."', employee_count: 20, tech_team_size: 20, website: 'https://www.humanapi.co/',logo: 'https://pbs.twimg.com/profile_images/378800000816901251/0dab64e7bd2cc59049520d7ca4649462_400x400.png', twitter: 'human_api', published: false
+  }
+]
+organization_list.each do |org|
+  Organization.create( org )
+  puts "Creating #{org[:name]}"
+end
 
 ClaimRequest.create(user_id: User.first.id, organization_id: Organization.first.id)
 ClaimRequest.create(user_id: User.last.id, organization_id: Organization.last.id)
+puts "Created Claim Request"
```

Comment:
Minor fix, please say `Created 2 Claim Requests` Also add a print to display how many users and how many organizations have been created.